### ISSUE TITLE
Index TECHMD datastreams coming from 7.x

### ISSUE THREAD

**Issue by mjordan** [Islandora/documentation#1246]
Since CrayFITS and the accompanying Islandora module are soon to be bundled with Islandora, I thought I'd ask this question here.

Sites migrating from 7.x will already have a TECHMD datastream, which is attached to the destination 8.x node as a media, like all 7.x datastreams. It would be great to be able to avoid regenerating FITS for objects migrated into 8.x and reuse the incoming TECHMD datastreams. To do this, we should figure out how we can use Islandora FITS to index incoming TECHMD datastreams the same way that it indexes FITS XML generated by CrayFITS. I'm happy to work on this.

**Comment by dannylamb** [comment]
If you have `islandora_fits` module enabled, and you tag the FITS file with the correct media use term on the way in, it'll automatically get sliced up into fields like something coming from Crayfits.  It would be good to install `islandora_fits` and then run our migrate tools to verify, but it should be automagic.

**Comment by mjordan** [comment]
Yeah, that was more or less the conclusion we came to yesterday during the call. I can test that, eventually.

**Comment by mjordan** [comment]
I will test this.

**Comment by mjordan** [comment]
TECHMD datastreams migrated from 7.x are not getting indexed. I notice that the Media Type of the migrated file is "File", not "METS Technical Metadata". Would that make a difference?

![migrated_media](https://user-images.githubusercontent.com/403918/73765961-2dc05780-472a-11ea-864c-7ad95ad58faf.png)


Also, I notice that there are now two separate "FITS File" Media Use tags (TID 1 and 34 on my Vagrant), which doesn't sound right.

@ajstanley any suggestions?

BTW if you don't disable the "Technical Metadata on Ingest" Context during my migration, you get two FITS media (one that is migrated, and the one that is generated by this Context). This probably should be documented somewhere. I can open a PR for that against https://github.com/Islandora-devops/migrate_7x_claw .

**Comment by dannylamb** [comment]
@mjordan Yes, that needs to come over as the FITS media type and not file.  And one of those FITS terms comes from migrate_7x_claw: https://github.com/Islandora-Devops/migrate_7x_claw/blob/master/migrate/tags.csv and we can axe it.

**Comment by dannylamb** [comment]
Looks like we decide what media type to make based on mimetype alone, which is a bit problematic here https://github.com/Islandora-Devops/migrate_7x_claw/blob/master/modules/islandora_migrate_7x_claw_feature/config/install/migrate_plus.migration.islandora_media.yml#L150

We'll need to take the DSID/tag into consideration too.

**Comment by dannylamb** [comment]
I think it can just be chained with another static map. Might have to declare a temp variable but I'm sure it can be done.  I'll give it a shot when I get a fresh environment up.
